app-frontend/src/app/components/common/listItemWidgets/listItemStatus/index.html

```diff
@@ -0,0 +1,14 @@
+<i
+    ng-repeat="eachStatus in $ctrl.updatedStatuses track by $index"
+    ng-class="$ctrl.statusMap[eachStatus.status].class"
+    ng-if="eachStatus.count"
+>
+    {{$ctrl.statusMap[eachStatus.status].text}}
+    <span ng-if="$ctrl.statusMap.objectType === 'upload'" class="right-margin-1">
```

Comment:
I think it makes more sense just to use ng-if="eachStatus.count". 

I'd also advocate for not mixing multiple concepts into the statusMap. Looking at the name, I'd assume that it only contains statuses, but you've added a separate type of metadata to it in a way that isn't separated in any way other than naming convention.